src/pages/index.js

```diff
@@ -125,6 +126,10 @@ function Nft(props) {
             {t('owner')}:&nbsp;{t('unclaimed')}
           </Button>
         )}
+        <Button onClick={() => copyLinkToNFT(props.developerId)}>
+          <LinkIcon />
+          &nbsp;Copy link to this NFT
+        </Button>
```

Comment:
`Button` has a `leftIcon` prop that makes this nice and easy :)

```suggestion
        <Button onClick={() => copyLinkToNFT(props.developerId)} leftIcon={<LinkIcon />}>
          Copy link to this NFT
        </Button>
```